CHANGELOG.md

```diff
@@ -13,6 +14,44 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
+# r1.2
+
+## Release Notes
+
+This public release contains the definition and documentation of
+* qos-booking v0.1.0
+* qos-booking-and-assignment v0.1.0
+
+The API definition(s) are based on
+* Commonalities v0.6.0 (r3.3)
+* Identity and Consent Management v0.4.0 (r3.3)
+
+## qos-booking v0.1.0
+
+**qos-booking v0.1.0 is the first initial release of this new API.**
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QoSBooking/r1.2/code/API_definitions/qos-booking.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QoSBooking/r1.2/code/API_definitions/qos-booking.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QoSBooking/blob/r1.2/code/API_definitions/qos-booking.yaml)
+
+### Added
+* Initial contribution of the QoS Booking API <br> which includes all alignments and fixes since the pre-release 
```

Comment:
You need to list all the completed PRs from M3 and M4 here

Reply:
Nope, only the changes compared to the previous **public release** - and as this is non-existing, everything is new.